composer.json

```diff
@@ -39,7 +39,8 @@
     "require-dev": {
         "fzaninotto/faker": "^1.5",
         "graham-campbell/testbench-core": "^1.0",
-        "phpunit/phpunit": "^4.7.6"
+        "phpunit/phpunit": "^4.7.6",
+        "laravel/homestead": "dev-master"
```

Comment:
Read through this: http://laravel.com/docs/5.1/homestead#per-project-installation

It is a new feature of Homestead.

I can change it to be a specific version though.